kotlintest-assertions/src/main/kotlin/io/kotlintest/inspectors/funcs.kt

```diff
@@ -16,22 +16,29 @@ fun <T> runTests(col: Collection<T>, f: (T) -> Unit): List<ElementResult<T>> {
 }
 
 fun <T> buildAssertionError(msg: String, results: List<ElementResult<T>>): String {
-
   val passed = results.filterIsInstance<ElementPass<T>>()
   val failed = results.filterIsInstance<ElementFail<T>>()
+  val maxOutputResult = 10
```

Comment:
This definitely should be a configurable amount, perhaps via SystemProperties

Reply:
Thank you your review!
Does below commit match with your intention?
8e5d3a4